src/odemis/gui/cont/tabs.py

```diff
@@ -938,9 +952,14 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 (panel.vp_sparc_bl, panel.lbl_sparc_view_bl)),
             (
                 panel.btn_sparc_view_br,
-                (panel.vp_sparc_br, panel.lbl_sparc_view_br)),
+                (panel.vp_sparc_br, panel.lbl_sparc_view_br)),  # default is now monochromator
         ])
 
+        # hack to overwrite the view buttons
+        if main_data.streak_ccd:
+            # Note: for now either streak camera HW or monochromator HW possible
```

Comment:
Maybe just move this comment into a larger comment before the vpv definition which states what's currently the behaviour with the monochromator and the streak_ccd.